Directory.Build.Props

```diff
@@ -49,6 +49,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <WarningLevel>Level4</WarningLevel>
+      <TreatWarningAsError>true</TreatWarningAsError>
```

Comment:
I see there is a `Directory.Build.Props` at the root of the repo that has a `ClCompile` element. it this can go there and then these files can remove the `WarningLevle` and `TreadWarningAsErrors`

```xml
    <ClCompile>
      <WarningLevel>Level4</WarningLevel>
```